uber/templates/registration/checkin.html

```diff
@@ -0,0 +1,120 @@
+<script type="text/javascript">
+    $(function() {
+        $('.dialog-form').dialog({ autoOpen: false });
+
+        $('.attendee-checkin').button().on('click', function () {
+            var attendee_id = $(this).attr('data-attendeeid');
+            var dialog = $('#' + attendee_id).dialog({
+                autoOpen: false,
+                height: 450,
+                width: 500,
+                modal: true,
+                buttons: [{
+                    text: 'Save & Check In',
+                    click: function() {
+                        check_in(attendee_id);
+                        dialog.dialog('close');
+                    }
+                }, {
+                    text: 'Cancel',
+                    click: function () {
+                        dialog.dialog('close');
+                    }
+                }],
+                // FIXME: OMG TERRIBLE POSITION HAX
+                open: function () {
+                  $('.ui-dialog').css('position', 'fixed')
+                    .css('top', '5%')
+                    .css('left', '33%');
+                }
+            });
+            dialog.find('input.num').keypress(function (event) {
+                if (event.keyCode === 13) {  // Enter key
+                    event.preventDefault();
+                    dialog.dialog('close');
+                }
+            });
+            dialog.dialog('open');
+        });
+    });
+
+    var check_in = function (id) {
+        if ($('#age_' + id).val() == {{ AGE_UNKNOWN }}) {
+            alert('You may not check someone in without confirming their age.');
+            return;
+        }
+        $.post('check_in', {
+            id: id,
+            csrf_token: csrf_token,
+            age_group:  $('#age_' + id).val(),
+            badge_num:  $('#num_' + id).val(),
+            group: $('#group_' + id).val() || ''
+        }, function (json) {
+            var message = json.message;
+            if (json.success) {
+                message += ' &nbsp; <a href="#" onClick="undoCheckin(\'' + id + '\', ' + json.pre_badge + ') ; return false">Undo</a>';
```

Comment:
The other problem with ``undoCheckin`` was that we weren't quoting the ``id`` parameter, which we didn't used to need to do.  But since we use uuids now, they need to be quoted.